doc/nrf/ug_nrf9160.rst

```diff
@@ -3,27 +3,23 @@
 Working with nRF9160
 ####################
 
-The |NCS| provides support for developing on the nRF9160 System in Package
-(SiP) using the nRF9160 DK (PCA10090), and offers samples
-dedicated for this device.
+The |NCS| provides support for developing on the nRF9160 System in Package (SiP) using the nRF9160 DK (PCA10090), and offers samples dedicated to this device.
```

Comment:
Maybe add a section tag to "Available drivers, libraries, and samples" and link to that section from "offers samples"?

Reply:
fixed